src/sentry/scripts/quotas/is_rate_limited.lua

```diff
@@ -1,28 +1,29 @@
 -- Check a collection of quota counters to identify if an item should be rate
 -- limited. Values provided as ``KEYS`` specify the keys of the counters to
 -- check, and values provided as ``ARGV`` specify the maximum value (quota
--- limit) and expiration time for each key.
+-- limit), expiration time for each key, and a key for the corresponding
+-- counter to subtract.
 --
 -- For example, to check a quota ``foo`` that has a limit of 10 items and
 -- expires at the Unix timestamp ``100``, as well as a quota ``bar`` that has a
 -- limit of 20 items and should expire at the Unix timestamp ``100``, the
 -- ``KEYS`` and ``ARGV`` values would be as follows:
 --
 --   KEYS = {"foo", "bar"}
---   ARGV = {10, 100, 20, 100}
+--   ARGV = {10, 100, "baz", 20, 100, "foo"}
```

Comment:
Since we're passing the "refund" keys as a part of `ARGV`, what's making sure they are on the same shard? I thought `KEYS` was used for routing. Maybe I'm missing something though, just want to make sure we're aware that these have to exist on the same shard.

Reply:
I was going to make this same comment as well — keys should be part of `KEYS` and not `ARGV` here for consistency. This script is executed via rb though, and is explicitly targeted to a host so data placement isn't an issue (ignoring Redis Cluster behavior for now.)

Reply:
i was assuming that `self.cluster.get_local_client_for_key` and passing in organization id was ensuring it was in the same shard. is that correct? I can update to make them part of `KEYS`